.woodpecker.yaml

```diff
@@ -0,0 +1,6 @@
+steps:
+  - name: deploy
+    image: alpine
+    commands:
+      - curl -L https://fly.io/install.sh | sh
```

Comment:
Should we track the version of fly?

Reply:
not really. because I saw errors in the forums people reported, until support tell them: did you update the cli?

the answer:
> No. it was an old version. I've updated now and it works. good bye.


Best practice are evil. as they don't make us think.
That's why CircleCI needs to be reviewed, not obeyed necessarily. Half the error I went through were invalid, or more. or just undesirable. pointless.

In this case setting a version is hassle with no benefit. the platform evolves with the CLI. 
if it breaks interfaces it better breaks the pipeline at  the command time than executing things with an old version.

Reply:
Okay so it depends on the use cases, am I right?

Reply:
Yes.

what benefit would it be to stick to a specific version of that cli?
what problems would it introduce if we stick to a version.

It falls down to the discussion we had. I mentioned maybe the OS (and some libs) could just grab the latest. If they are stable. 

woodpecker is likely not stable. Radicle as we've seen is not stable. There is far more chance they break it with a release than fix critical vulnerabilities without damaging stuff. OS distribs tend to do a good job. So we tend to update all (except maybe openssl!)

Reply:
Amazing explanation! I should be more conscious of doing version tracking from now.  Thank you for your insights!